lib/console.ex

```diff
@@ -1,29 +1,62 @@
 defmodule Console do
-  def print_board(board) do
-    IO.write """
 
-    --------------
 
-      #{Enum.at(board,0)}   #{Enum.at(board, 1)}   #{Enum.at(board, 2)}
-     ===+===+=== 
-      #{Enum.at(board, 3)}   #{Enum.at(board, 4)}   #{Enum.at(board, 5)}
-     ===+===+=== 
-      #{Enum.at(board, 6)}   #{Enum.at(board, 7)}   #{Enum.at(board, 8)}
-
-    --------------
+  def print_board(game) do
+    board = game.board
+    case game.size do 
+      4 ->
+        IO.write """
+      
+        ---------------------------
+    
+             #{Enum.at(board, 0)}   #{Enum.at(board,1)}   #{Enum.at(board,2)}   #{Enum.at(board, 3)}
+            ===+===+===+===
+             #{Enum.at(board, 4)}   #{Enum.at(board, 5)}   #{Enum.at(board, 6)}   #{Enum.at(board, 7)}
+            ===+===+===+===
+             #{Enum.at(board, 8)}   #{Enum.at(board, 9)}   #{Enum.at(board, 10)}   #{Enum.at(board, 11)}
+            ===+===+===+===
+             #{Enum.at(board, 12)}   #{Enum.at(board, 13)}   #{Enum.at(board, 14)}   #{Enum.at(board, 15)}
     
-    """
+        ---------------------------
+    
+        """
+      3 ->
+        IO.write """
+      
+        --------------
+      
+          #{Enum.at(board,  0)}   #{Enum.at(board, 1)}   #{Enum.at(board, 2)}
+         ===+===+=== 
+          #{Enum.at(board, 3)}   #{Enum.at(board, 4)}   #{Enum.at(board, 5)}
+         ===+===+=== 
+          #{Enum.at(board, 6)}   #{Enum.at(board, 7)}   #{Enum.at(board, 8)}
+      
+        --------------
+        
+        """
+    end
   end
 
+  # def print_board(board) do
```

Comment:
Do we need the comment to stay?  Odds are good this will resurface on its own if we need it, right?